### PR TITLE
Improving documentation

### PR DESCRIPTION
**Pull request by nellshamrell**
Expanding and adding in more specific steps around problems I found when setting up the local dev environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ repositories are:
 ## Requirements
 
 - Ruby 2.1.3
-- PostgreSQL 9.3+
+- PostgreSQL 9.2
```
Comment (raskchanky) [inline README.md]: It might be worth changing this to `9.2+`.  9.3 does work, as that's what I've been using.

**Comment by raskchanky** [comment]
:+1:

**Comment by nellshamrell** [comment]
Confirmed that these instructions work on a VM with a fresh OS X Yosemite install.  Marking this as ready,

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Pull request merged**